lib/interceptor.js

```diff
@@ -102,7 +102,10 @@ module.exports = class Interceptor {
   replyWithError(errorMessage) {
     this.errorMessage = errorMessage
 
-    _.defaults(this.options, this.scope.scopeOptions)
+    this.options = {
+      ...this.scope.scopeOptions,
+      ...this.options,
+    }
```

Comment:
Are these guaranteed to be shallow objects?

Reply:
I’m not sure, though I think [this is the same behavior as _.defaults](https://lodash.com/docs/#defaults).

Reply:
yes, your change is parity. 
But I'm questioning if that is correct. Some of the options are objects, eg `reqheaders`. So I think it's a bug that it doesn't do deep cloning.

Reply:
That's possible. I don't know what the use cases are for merging these options. I'm not sure if it's possible that reqheaders could end up in both (or if merging them would be the correct thing to do). Would you like me to add a todo comment?

Reply:
hmmm, yeah it's a weird undocumented case. I think I'd vote for keeping with your current change since it keeps parity and come back to it again if anyone ever opens an issue.